lib/msal-node/src/client/ClientCredentialClient.ts

```diff
@@ -229,6 +229,9 @@ export class ClientCredentialClient extends BaseClient {
                 authority.tokenEndpoint,
                 queryParametersString
             );
+
+            this.logger.info("Sending token request to host: " + authority.tokenEndpointHost);
```

Comment:
- Can we just print out the full tokenEndpoint, instead of just the host? Maybe helpful to have full string for debugging? Unless this is enough?
- Can this log be put near the actual call, like in `executePostToTokenEndpoint`? Unless we already log stuff there.